src/extension/content-script/batteries/Mastodon.ts

```diff
@@ -0,0 +1,65 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher =
+  /^https:\/\/(bitcoinhackers\.org|kosmos\.social)\/(web\/)?@\S+/;
```

Comment:
Is there no way to generalize this?

Reply:
it could check the actual HTML code, but we wanted to avoid this initially. 
Do you think it's OK to do that?